### PR TITLE
Fixed Issue-2 by adding dynamic entrypoint.

### PR DESCRIPTION
**Pull request by SarahKef**
Implemented Issue #2

### CODE REVIEW

File: webwrapper/server.py [hunk 1]
```diff
@@ -19,12 +20,19 @@ def _setup_framework(project_config):
...
+        if name is None:
+            name = entrypoint
 
-        if path is None:
-            path = name
         func = getattr(app_module, entrypoint)
+
+        if path is None:
+            if 'GET' in methods:
+                path_builder.build_path(func, name)
```
Comment (hammadk373) [inline webwrapper/server.py]: missing assignment back to `path`

File: webwrapper/utils/path_builder.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        base_name = func.__name__
+    if param_kinds is None:
+        param_kinds = [Parameter.POSITIONAL_OR_KEYWORD, Parameter.KEYWORD_ONLY]
+
+    path = base_name
+    sig = signature(func)
+    for param in sig.parameters.values():
+        if (param.kind in param_kinds and
+                param.default is param.empty):
+            path = path + "/{{{0}}}".format(param.name)
```
Comment (hammadk373) [inline webwrapper/utils/path_builder.py]: fastAPI support assigning type to the path params as well. for example:
path/{name:str}/{age:int}

the type of the arguments are available from `param.annotation` attribute, we want to include those in the path is the function defines them.

that will allow us to differentiate between overloaded endpoints that take different types of arguments. for example:
path/{name:str}/{age:int}
and
path/{name:str}/{age:str}

**Review by hammadk373 — CHANGES_REQUESTED** [review]

File: webwrapper/server.py (hunk 1 above)
Comment (SarahKef) [reply webwrapper/server.py]: done

File: webwrapper/utils/path_builder.py (hunk 2 above)
Comment (SarahKef) [reply webwrapper/utils/path_builder.py]: done

**Review by hammadk373 — APPROVED** [review]

**Pull request merged**